test/log-join-concurrent.spec.js

```diff
@@ -37,6 +37,8 @@ Object.keys(testAPIs).forEach(IPFS => {
 
     after(async () => {
       await stopIpfs(ipfs)
+      console.log(testIdentity)
+      await testIdentity._provider._keystore._store.close()
```

Comment:
This feels a bit weird (but admittedly I don't have the full context) as generally properties that are prefixed with `_` (underscore) are intended to be private (as far as private properties/variables go in JS). Is there a way to call `close()` "directly", eg. `testIdentity.provider.keystore.store.close()`? If not, what would it require to do so, instead of access the "private" variables directly?

Reply:
We could either add a keystore getter to the provider or add a `close()` to the provider which calls `this._keystore.close()`, then this can be replaced with `testIdentity.provider.close()`, I would prefer adding a `close()` to the provider